.github/workflows/v_apps_and_modules_compile_ci.yml

```diff
@@ -53,10 +53,10 @@ jobs:
 
       - name: Test discord.v
         run: |
-          echo "Clone https://github.com/DarpHome/discord.v/"
-          .github/workflows/retry.sh git clone https://github.com/DarpHome/discord.v/ discord && cd discord
-          echo "Checkout last known good commit"
-          git checkout 533485c08f21df91ff62fea9477e7017d21f91c4
+          echo "Clone https://github.com/DarpHome/discord.v"
+          .github/workflows/retry.sh v install https://github.com/ttytm/discord.v@temp/v-ci && cd ~/.vmodules/discord
```

Comment:
https://github.com/DarpHome/discord.v/ was updated, and now has 789de8ad35ee2683fbcd950fc07936f052088d0c as well.

Reply:
Yes, unfortunately I missed one thing in the PR. The commit would not work yet. It would require to change one more field to pass. I'll submit another PR soon.